spec/markdown-preview-view-spec.coffee

```diff
@@ -210,6 +210,31 @@ describe "MarkdownPreviewView", ->
         expect(fs.isFileSync(outputPath)).toBe true
         expect(atom.workspace.getActiveTextEditor().getText()).toBe expectedOutput
 
+    describe "text editor style extraction", ->
+
+      disposableStyleSheet = disposableStyleSheet2 = extractedStyles = null
```

Comment:
The style we typically employ in this situation is:

```coffee
[disposableStyleSheet, disposableStyleSheet2, extractedStyles] = []
```

Would be good to match that for consistency.